src/components/UserCard.js

```diff
@@ -1,7 +1,37 @@
 import React, { useState } from 'react'
 import CopyToClipboard from 'react-copy-to-clipboard'
+import axios from 'axios'
+
 
 function UserCard(props) {
+    const handleStatus = (e) => {
+        const target = e.target
+        setDonated({status: target.checked})
+        console.log(target.checked)
+    }
+
+    function thanksRandomizer() {
+        const thanks = ['Thank you so much!🙏', 'Amazing! 🕺🏿', 'That\'s rad! 👍🏿', 'Thank you! ❤️', 'You\'re a lifesaver! 🤗', 'The world is lucky to have you in it! 🏄‍♂️🏄‍♂️']
+        const randomthanks = thanks[Math.floor(Math.random() * thanks.length)]
+        return randomthanks
+        
+    }
+
+    
+    async function postDonation(){
+    const response = await axios.put(
+        `https://good-faith.herokuapp.com/api/people/29`,
+        { donationAmount: "1000.00" },
```

Comment:
@vunderkind I think you omitted making these request params dynamic

Reply:
- `donationAmount` should get what's in the input field
- request endpoints should reference the dynamic "recipients" id